spec/system/tech_shop_spec.rb

```diff
@@ -37,150 +39,142 @@
     # Subproducts
     @camera = Camera.create!(name: 'canon eos rebel t3', href: 'canon')
 
-    100.times { Product.create!(name: 'crapoola', href: 'crappy', tags: ['crappy']) }
```

Comment:
This alone removed over half a second from the test runtime for me. These 100 "crapoola"s are actually used in the specs.